config/main.py

```diff
@@ -548,7 +575,12 @@ def shutdown(interface_name, verbose):
 def startup(interface_name, verbose):
     """Start up interface"""
     if get_interface_mode() == "alias": 
-        interface_name = interface_alias(interface_name)
+        default_name = interface_alias_to_name(interface_name)
```

Comment:
Please refactor this code in the same manner as I suggested in my comment on line 411.